web/widgets.py

```
@@ -135,3 +139,224 @@ def __init__(self, attrs=None):
         if attrs:
             default_attrs.update(attrs)
         super().__init__(default_attrs)
+
+
+class TailwindJSONWidget(forms.Widget):
+    """A custom widget for editing JSON arrays of strings as a list of inputs with drag-and-drop reordering."""
+
+    def __init__(self, attrs: Dict = None) -> None:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)